TheATeam/AvengersTeam/src/App.js

```diff
@@ -63,12 +72,33 @@ const AppLayout = () => {
     return (
         <div>
             <HeadingComponent />
-            <SearchPageComponent />
+            <Outlet />
         </div>
     )
 }
 
+const appRoot = createBrowserRouter([
+    {
+        path: "/",
+        element: <AppLayout />,
+        errorElement: <ErrorComponent />,
+        children: [
```

Comment:
try adding path for default as well..
Refer ---> https://reactrouter.com/en/main/start/tutorial#index-routes